components/Article/Codebox/__tests__/index.test.tsx

```diff
@@ -0,0 +1,99 @@
+import userEvent from '@testing-library/user-event';
+import { render, screen } from '@testing-library/react';
+import { IntlProvider } from 'react-intl';
+
+import Codebox, { replaceLabelLanguages, replaceLanguages } from '../index';
+
+jest.mock('isomorphic-dompurify', () => {
+  return {
+    sanitize: jest.fn().mockImplementation(source => source),
+  };
+});
+
+describe('Replacer tests', (): void => {
+  it('replaceLabelLanguages', (): void => {
+    expect(replaceLabelLanguages('language-console')).toBe('language-bash');
+  });
+
+  it('replaceLanguages', (): void => {
+    expect(replaceLanguages('language-mjs')).toBe('language-js');
+    expect(replaceLanguages('language-cjs')).toBe('language-js');
+    expect(replaceLanguages('language-javascript')).toBe('language-js');
+    expect(replaceLanguages('language-console')).toBe('language-bash');
+    expect(replaceLanguages('language-shell')).toBe('language-bash');
+  });
+});
+
+describe('Codebox component (one lang)', (): void => {
+  const code = 'const a = 1;';
+
+  it('renders correctly', (): void => {
+    const { container } = render(
+      <IntlProvider locale="en" onError={() => {}}>
+        <Codebox>
+          <pre className="language-js">{code}</pre>
+        </Codebox>
+      </IntlProvider>
+    );
+    expect(container).toMatchSnapshot();
+  });
+
+  it('should copy content', async () => {
+    const user = userEvent.setup();
+
+    render(
+      <IntlProvider locale="en" onError={() => {}}>
+        <Codebox>
+          <pre className="language-js">{code}</pre>
+        </Codebox>
+      </IntlProvider>
+    );
+
+    const navigatorClipboardWriteTextSpy = jest.spyOn(
+      navigator.clipboard,
+      'writeText'
+    );
+
+    const buttonElement = screen.getByText('components.codeBox.copy');
+    await user.click(buttonElement);
+
+    expect(navigatorClipboardWriteTextSpy).toHaveBeenCalledTimes(1);
+    expect(navigatorClipboardWriteTextSpy).toHaveBeenCalledWith(code);
+  });
+});
+
+describe('Codebox component (multiple langs)', (): void => {
+  const code = `const http = require('http');
+--------------
+import http from 'http';`;
+
+  it('renders correctly', (): void => {
```

Comment:
@araujogui this test can be removed, as Storybooks now have their own snapshot testing. You can also remove this snapshot,

Don't forget to run `npm run test:storybook:snapshot` just to double check all Storybooks have Snapshots